### PR TITLE
dockerised webiste

### PR DESCRIPTION
**Pull request by mdothman**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <meta property="og:url" content="{{ page.url | absolute_url }}" />
+  <meta property="og:type" content="website" />
+  <meta property="og:title" content="{% if page.title %}{{ page.title | escape }} by {% endif %}{{ site.title | escape}}" />
+  <meta property="og:description" content="{% if page.description %}{{ page.description | strip_html | strip_newlines | truncate: 400 }}{% else %}{{ site.description }}{% endif %}" />
+  <meta property="og:image" content="{% if page.image %}{{ page.image | absolute_url }}{% else%}{{'/assets/images/logos/hfla_fb_logo.png' | absolute_url }}{% endif %}" />
```
Comment (cnk) [inline _includes/head.html:9]: Copy paste. /assets/images/logos/ does not exist. Again, would be good to steal the logo from the organization: https://github.com/100Automations

File: _includes/head.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  <title>{% if page.title %}{{ page.title | escape }} - {% endif %}{{ site.title | escape}}</title>
+  <meta name="description"
+    content="{% if page.description %}{{ page.description | strip_html | strip_newlines | truncate: 400 }}{% else %}{{ site.description }}{% endif %}" />
+  <link rel="canonical" href="{{ page.url | absolute_url }}" />
+
```
Comment (cnk) [inline _includes/head.html:15]: Please remove the extra lines before </head>

**Review by cnk — COMMENTED** [review]
Good start. The project builds in docker and serves the README.md as the index page. 

One addition to the README page could be instructions to clone this repo and run `docker-compose up` to see site as you edit.

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+lsi: false
+safe: true
+
+source: .
+
+title: 100 Automations
+email: hello@hackforla.org
+description: Can you design, write, or code? Are you an activist with an idea? You can help Los Angeles live up to its potential at Hack for LA. Everyone is welcome!
```
[the remark this answers is not in the document]
Comment (mdothman) [reply _config.yml:8]: Thank you for that. I think having an accurate description is very helpful for the user.

**Comment by cnk** [comment]
Looks good. This sets up the basics for developing a Jekyll site inside a Docker container. Great start!

**Comment by mdothman** [comment]
Thank you for your help 💯

**Pull request merged**